TestApp/update-npm-packages.sh

```diff
@@ -24,5 +24,8 @@ npm install
 echo "Build shared framework..."
 (cd ../AppCenterReactNativeShared/ios && SRCROOT=`pwd` ./build-fat-framework.sh)
 
-echo "Running jetify"
+echo "Running jetify to resolve AndroidX compatibility issues..."
 npx jetify
+
+echo "Running pod install..."
+(cd ios && pod install)
```

Comment:
Note the new development workflow is to run `update-npm-packages.sh` script from TestApp directory.

With autolinking, `TestApp/update-link-to-appcenter.sh` script is no longer needed, hence deleted in this PR.